icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2041,6 +2044,45 @@ class U_I18N_API NumberFormatterSettings {
      */
     Derived scale(const Scale &scale) &&;
 
+    /**
+     * Specifies the usage of the unit ("person", "road", "person" ...etc.)
+     *
+     * NOTE:    `usage` will change the output unit depending on the `Locale`  
+     *          and the unit value.
+     *          For Example:
+     *              Locale: en_US
```

Comment:
All the spaces will make this render as a code block in Markdown.  Please write this as normal Markdown.  To preview, you can run `make doc` (or `make docs`?).